cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -110,6 +110,7 @@ public static void CopyTo(this byte[] source, int sourceIndex, IBuffer destinati
         public static byte[] ToArray(this IBuffer source)
         {
             if (source == null) throw new ArgumentNullException(nameof(source));
+            if (source.Capacity == 0) return Array.Empty<byte>();
```

Comment:
capacity or length?

Reply:
Buffers can only be constructed with a capacity, so I think the idea is that empty buffers should become empty arrays...

Reply:
I would assume a buffer with a capacity of 2 would result in a ToArray of 0 length.

[This](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.windowsruntime.windowsruntimebufferextensions.toarray?view=dotnet-plat-ext-3.1#System_Runtime_InteropServices_WindowsRuntime_WindowsRuntimeBufferExtensions_ToArray_Windows_Storage_Streams_IBuffer_) was the previous documentation for this API which also indicates similarly.